### PR TITLE
sort-time should not be the basis of any updates

### PR DESCRIPTION
**Pull request by erwinmombay**
also cleaned up tests.

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce PTAL

File: extensions/amp-live-list/0.1/amp-live-list.js [hunk 1]
```diff
@@ -265,6 +265,7 @@ export class AmpLiveList extends AMP.BaseElement {
           this.updateTime_ = updateTime;
         }
         updates.push(orphan);
+        continue;
```
Comment (cramforce) [inline extensions/amp-live-list/0.1/amp-live-list.js]: Maybe use `else if` and no `continue`?

Comment (erwinmombay) [inline extensions/amp-live-list/0.1/amp-live-list.js]: will do, they are exclusive so makes more sense to do that

Comment (erwinmombay) [inline extensions/amp-live-list/0.1/amp-live-list.js]: done.

**Comment by erwinmombay** [comment]
@cramforce updates done.

**Comment by cramforce** [comment]
LGTM

**Pull request merged**